README.md

```diff
@@ -47,3 +47,44 @@ print(get_speedify_settings())
 ```
 
 For the full list of settings, build the sphinx documentation by running make html in the docs/ folder.
+
+## Changelog
+
+### Release 12.5.x
+
+Added
+  - `dns(str)`
+  - `streamtest()`
+  - `directory(str)`
+  - `gateway(str)`
+  - `esni(bool)`
+  - `headercompression(bool)`
+  - `privacy(str, bool)`
+  - `daemon(str)`
+  - `login_auto()`
+  - `login_oauth(token)`
+  - `streamingbypass_domains(str, str)`
+  - `streamingbypass_ports(str, str)`
+  - `streamingbypass_ipv4(str, str)`
+  - `streamingbypass_ipv6(str, str)`
+  - `streamingbypass_service(str, bool)`
```

Comment:
These functions are out of date with the recent changes.

Reply:
All changed (except the `streamingbypass_service(str, bool)` item)